acceptance/tests/validate_file_paths.rb

```diff
@@ -0,0 +1,106 @@
+test_name 'C93828 - pxp-agent - File paths from spec'
+
+# verify that the installed file paths match those of the
+# puppet specification.
+# https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md
+#
+
+# {{{ helper methods
+def config_options(host)
+  platform = host[:platform]
+  ruby_arch = host[:ruby_arch]
+  type =  @options[:type]
+
+  case platform
+  when /windows/
+    if platform =~ /2003/
+      common_app_data = 'C:/Documents and Settings/All Users/Application Data'
+    else
+      common_app_data = 'C:/ProgramData'
+    end
+    if ruby_arch == 'x64'
+      ruby_arch = /-64/
+    else
+      ruby_arch = /-32/
+    end
+    if platform =~ ruby_arch
```

Comment:
My understanding is that if you are running a 32bit Windows, you are only going to have "Program Files" since you cannot install 64bit software on it. So, the "Program Files (x86)" directory is used when the ruby is 32bit, but the OS is 64bit.